docs/plugin-proposal-pipeline-operator.md

```diff
@@ -22,31 +22,60 @@ $ npm install --save-dev @babel/plugin-proposal-pipeline-operator
 }
 ```
 
-The Pipeline Proposal is one of three competing implementations. Which implementation the plugin should use is configured with the `"proposal"` option. This option is required and should be one of:
+The pipeline proposal is one of three competing implementations. Configure with proposal to use with the required `"proposal"` option.
 
-- `"minimal"` – [Minimal Pipeline](https://github.com/tc39/proposal-pipeline-operator/)
-- `"smart"` - [Smart Pipeline](https://github.com/js-choi/proposal-smart-pipelines) - Added in `v7.3.0`
-- `"fsharp"` - [F#-Style Pipeline](https://github.com/valtech-nyc/proposal-fsharp-pipelines) - Added in `v7.5.0`
+| Value | Proposal | Version added |
+| ----- | -------- | ------------- |
+| `"minimal"` | [Minimal F#-style pipes](https://github.com/tc39/proposal-pipeline-operator/) | `v7.0.0`
+| `"fsharp"` | [F#-style pipes with `await`](https://github.com/valtech-nyc/proposal-fsharp-pipelines) | `v7.5.0`
+| `"hack"` | [Hack-style pipes](https://github.com/js-choi/proposal-hack-pipes) | `v7.15.0`
+| `"smart"` | [Smart-mix pipes](https://github.com/js-choi/proposal-smart-pipelines) (deprecated) | `v7.3.0`
 
-When one of the implementations is accepted, it will become the default and the `"proposal"` option will no longer be required.
+If `"proposal": "hack"` is used, then a `"topicToken": "#"` option must also be included. In the future, other choices for `topicToken` will be added.
+
+The `"proposal": "smart"` option is deprecated and subject to removal in a future major version.
```

Comment:
Q: is someone still working on the `minimal` proposal?

Reply:
Yeah, the minimal-F# proposal is kind of in stasis but is still viable for TC39. It is basically backed by the champion(s) for all pipe proposals (@TabAtkins and maybe @ljharb), with its specification in the original proposal repository [tc39/proposal-pipeline-operator](https://github.com/tc39/proposal-pipeline-operator).